README.md

```diff
@@ -1 +1,59 @@
-# ODP
\ No newline at end of file
+# E2E-Test Framework
+
+##1. How to run the test against a) Local sandbox and b) Remote CDAP instance.
+
+### Local sandbox:
+
+Provide the local sandbox URL in [ConnectionParameters.properties](src/main/resources/ConnectionParameters.properties) file.
+
+In the ConnectionParameters.properties file, provide the local sandbox URL in
+[ConnectionParameters.properties](src/main/resources/ConnectionParameters.properties)
+
+All the configuration details need to be present in this file specific to test.
+Here configuration parameters are defined to use everytime wherever required. Changes are done directly here if need to change any particular value. Below parameter values need to be changed to run the test against different sandbox.
+
+Property name and its value is CDFURL=http://localhost:11011/pipelines/ns/default/studio
+Project-ID=cdf-athena<br>
+@TC-Demo-1_GCS=gs://cdf-athena-test/test.csv<br>
+@TC-Demo-2_GCS=gs://cdf-athena-test/test-DataTypes.csv<br>
+Data-Set=test_automation<br>
+tableDemo=DemoCheck1<br>
+
```

Comment:
I don't think anything other than cdfUrl is specific to sandbox. @rmstar what do you think?

Reply:
Done.

Reply:
projectId config will need to be changed if the user wants to run the test on a different. They might not have access to cdf-athena.

Reply:
Done.